website/docs/faq.mdx

```diff
@@ -47,3 +47,23 @@
     <p>For details on the registry format, please refer to <a href="https://github.com/TabbyML/registry-tabby/blob/main/models.json">models.json</a></p>
   </div>
 </details>
+
+<details>
+  <summary>Does Tabby currently support arm64/v8 CPUs?</summary>
+  <div>
+    <p>We don't officially support it yet, but we do have an experimental feature that can be accessed using the Docker command.</p>
+    <p>For more context, see <a href="https://github.com/TabbyML/tabby/pull/1022/files">this pull request</a></p>
+    <p>To use this fix:</p>
+    <ol>
+      <li>Clone the tabby repo</li>
+      <li>Build the docker image locally with <code>--platform linux/amd64</code></li>
+      <li>Run the docker image</li>
+    </ol>
+    <pre>
+      <code>git clone https://github.com/TabbyML/tabby.git</code>
+      <code>cd tabby</code>
+      <code>docker build -t tabby:latest --platform linux/amd64 .</code>
+      <code>docker run -dt -p 8080:8080 -v $HOME/.tabby:/data --name tabby tabby:latest serve --model TabbyML/StarCoder-1B</code>
```

Comment:
i don't think this actually works on arm64 (with cuda), since it's built in linux/amd64, and started without passing nvidia devices